features/foutafhandeling.feature

```diff
@@ -62,3 +62,45 @@ Rule: min waarde mag niet groter zijn dan max waarde
         | path                   | query string                               | parameter   | code  | reason omschrijving              |
         | /adresseerbareobjecten | ?oppervlakte[min]=200&oppervlakte[max]=100 | oppervlakte | range | min mag niet hoger zijn dan max. |
         | /panden                | ?bouwjaar[min]=2000&bouwjaar[max]=1000     | bouwjaar    | range | min mag niet hoger zijn dan max. |
+
+Rule: min waarde mag niet kleiner zijn dan toegestane minimum waarde
+
+    Abstract scenario: opgegeven minimum waarde lager dan toegestane minimum waarde
+        Als '<path><query string>' wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | <detail>                                      |
+        En bevat de response de volgende invalidParams
+        | name        | code    | reason               |
+        | <parameter> | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path                   | query string                                                               | parameter        | code    | reason omschrijving            |
+        | /adresseerbareobjecten | '?bbox=134647,457842,135127,458307&oppervlakte[min]=0&oppervlakte[max]=200 | oppervlakte[min] | minimum | Waarde is lager dan minimum 1. |
```

Comment:
In de query string kolom beginnen alle query string waarden met een quote. Die kan weg